### PR TITLE
Feat/get category endpoint

### PR DESCRIPTION
**Pull request by fatjan**
This is to add a get categories endpoint

### CODE REVIEW

File: app/main/service/category_service.py [hunk 1]
```diff
@@ -17,3 +17,19 @@ def create_category(category_name):
...
+            return {"status": "success", "message": "No categories found", "data": []}, 200
+        
+        response_object = {
+            "status": "success",
+            "message": "Successfully get.",
+            "data": categories,
+        }
+        return response_object, 200
+    except  Exception as e:
+        log.error(f"Error in get_all_reviews: {str(e)}")
```
Comment (Achmadqizwini) [inline app/main/service/category_service.py:34]: it should be "category"

File: app/main/model/category.py [hunk 2]
```diff
@@ -39,3 +43,12 @@ def create_category(self, category_name):
             return self.serialize()
         except Exception as e:
             raise e
+    
+    def get_categories(self):
```
Comment (fahmirzldi) [inline app/main/model/category.py:47]: For consistency should this better be "get_all_categories"?

Comment (fatjan) [reply app/main/model/category.py]: done, kindly review it again, thank you

File: app/main/service/category_service.py (hunk 1 above)
Comment (fatjan) [reply app/main/service/category_service.py]: Thanks, I fixed the typo. Kindly review it again.

File: app/main/model/category.py [hunk 3]
```diff
@@ -31,6 +31,10 @@ def save(self):
 
     def create_category(self, category_name):
         try:
+            category = self.query.filter_by(name=category_name).first()
+            if category:
+                return category.serialize()
```
Comment (fahmirzldi) [inline app/main/model/category.py:36]: Interesting, so instead of throwing error, if the category is already made it will just return the category?

**Review by fahmirzldi — DISMISSED** [review]

File: app/main/model/category.py (hunk 3 above)
Comment (fatjan) [reply app/main/model/category.py:36]: yes

**Review by AchmadQiz — APPROVED** [review]

**Pull request merged**